Automattic/jetpack
# Fix WordAds defaulting to "house ad" mode on initial activation

#### Changes proposed in this Pull Request:
* Changes the way WordAds sets options to fix sites defaulting to "house ad" mode.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Spin up a new Jetpack site with a qualified plan
* Enable the Ads module
* Verify that the `wordads_house` option is set to the empty string `''` — i.e. `false`
